content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -166,11 +170,15 @@ to manage the lifecycle of Istio.
     56
     {{< /text >}}
 
-1. Install the `istio` chart:
+1. Install the `istio` chart in demo mode with mTLS enabled:
+
+{{< tip >}}
+This mode is designed to showcase Istio functionality with modest resource requirements. It enables high levels of tracing and access logging so it is not suitable for performance tests. You may use [values.yaml from the istio/tools repo](https://github.com/istio/tools/tree/master/perf/istio) to performance test Istio.
```

Comment:
Here and in the other place, change:

You may use this [streamlined `values.yaml`](https://github.com/istio/tools/tree/master/per/istio) file to test Istio's performance.